sdk/bcs/README.md

```diff
@@ -1,10 +1,13 @@
 # BCS - Binary Canonical Serialization
 
-This small and lightweight library implements [Binary Canonical Serialization (BCS)](https://github.com/zefchain/bcs) in JavaScript, making BCS available in both Browser and NodeJS environments.
```

Comment:
enabled `proseWrap` in prettier, because manually formatted line lengths in markdown drive me crazy